bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/WindowsDefenderConfigurator.java

```diff
@@ -76,11 +76,12 @@
 @EventTopics(UIEvents.UILifeCycle.APP_STARTUP_COMPLETE)
 public class WindowsDefenderConfigurator implements EventHandler {
```

Comment:
I would use E4 plugin here instead of OSGi as you can then inject everything you need and still get notified by events